workspaces/web/webpack.config.js

```diff
@@ -144,13 +144,20 @@ module.exports = {
         },
         mode: process.env.NODE_ENV,
         performance: {
-            hints: false, // it doesn't matter if server.js is large
```

Comment:
Well, it really kinda doesn't matter if server.js is large, at least not to the same degree that it matters that client.js is large.

I think you meant to introduce a limit on client.js, not server.js.

Reply:
I've changed the webpack config to limit only the client size.  Please check if it is what you was expecting.

Reply:
yes, it does make sense. How do the set limits compare to the actual numbers observed?